install/bootstrap_livecd_remote_iso.md

```diff
@@ -202,9 +232,9 @@ On first login (over SSH or at local console) the LiveCD will prompt the adminis
 
          ```bash
          pit# echo "
-         CSM_PATH=/var/www/ephemeral/${CSM_RELEASE}
          CSM_RELEASE=${CSM_RELEASE}
-         PITDATA=/var/www/ephemeral
+         PITDATA=$(lsblk -o MOUNTPOINT -nr /dev/disk/by-label/PITDATA)
+         CSM_PATH=${PITDATA}/${CSM_RELEASE}
```

Comment:
This won't work because the PITDATA variable isn't set. The line above it doesn't set it -- it just echoes the assignment to be written to the /etc/environment file. If you want to use the $PITDATA variable within this echo, it will need to be first set before this echo is done.

Reply:
I see.

I'll fix this up by setting it in the above step where the other 2 variables are set.